src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/UsecaseSelection.tsx

```diff
@@ -80,6 +80,7 @@ const UsecaseSelection = (props: { endTimeArray; setEndTimeArray; SREdata; setSR
     const response = await fetch(getListExperimentsURLWithParams(value));
     const data = await response.json();
     const experimentUsecase = data[0].experiment_usecase_type;
+    console.log(experimentUsecase);
     let usecase;
     if (experimentUsecase) {
       usecase = Object.keys(experimentUsecase).filter((key) => experimentUsecase[key] === true) + ' ';
```

Comment:
we have removed 'usecase' from the experiment_type. can u try to check if we still need the code on line no 84-89 which is trying to assign usecase variable some value.

Reply:
removed redundant 'experimentUsecase', as experiment type under workload details refers to the actual experiment_type i.e `container` or `namespace`